src/lib/components/NewLayeredMap/components/CompositeMapLayers.jsx

```diff
@@ -76,19 +76,41 @@ class CompositeMapLayer extends Component {
         ));
     }
     // add default bounds?
-    addImage = (image, layerGroup) => {
-        const bounds = image.bounds.map(xy => yx(xy));
-        if ("colormap" in image){
-            layerGroup.addLayer(L.imageWebGLOverlay(image.url, bounds, image.colormap, {
-                shader: image.shader
-            }));
+    addImage = (imageData, layerGroup) => {
```

Comment:
```suggestion
    addImage = (imageData) => {
```

trenger vel ikke layergroup her lenger